DataFormats/Detectors/CTP/include/DataFormatsCTP/Configuration.h

```diff
@@ -56,41 +64,48 @@ struct CTPDetector {
 struct CTPCluster {
   CTPCluster() = default;
   std::string name;
-  uint32_t detectorsMask;
-  std::vector<std::string> detectorsNames;
+  o2::detectors::DetID::mask_t getClusterDetMask() const;
+  std::vector<o2::detectors::DetID> detectors;
```

Comment:
in principle, instead of the std::vector<o2::detectors::DetID> you could simply use DetID::mask_t

Reply:
yes, fine. What is motivation ? We want to save memory ? Is it more elegant ?

Reply:
The size and serialization speed: if this goes to ccdb rather than to the data, then the size argument is irrelevant, but to send as a DPL message such object or to store it in the file you need to go via root serialization of every data member.
Anyway, this is a minor comment, if the class is for CCDB only.